spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -191,29 +191,6 @@
     expect(userinfo_response[:verified_at]).to eq(profile.verified_at.to_i)
   end
 
-  it 'returns a null verified_at if the account has not been proofed', driver: :mobile_rack_test do
-    token_response = sign_in_get_token_response(
-      scope: 'openid email profile:verified_at',
-      handoff_page_steps: proc do
-        expect(page).to have_content(t('help_text.requested_attributes.verified_at'))
-        expect(page).to have_content(t('help_text.requested_attributes.verified_at_blank'))
-
```

Comment:
I think we might want to keep this spec to make sure that the OIDC response contains the expected values. But we can update to remove the assertions in this block, since we're not expecting that content.